src/Microsoft.Diagnostics.Monitoring.EventPipe/Logs/LogRecordException.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.Diagnostics.Monitoring.EventPipe
+{
+    internal readonly struct LogRecordException : IEquatable<LogRecordException>
```

Comment:
I think a record struct might give you these semantics automatically.

Reply:
Updated